src/OXRS_LCD.cpp

```diff
@@ -147,8 +147,8 @@ void OXRS_LCD::show_MQTT_topic (char * topic)
   sprintf(buffer, "MQTT: %s",topic);
   tft.drawString(buffer, 12, 80);
 
-  _set_mqtt_rx_led(0);
-  _set_mqtt_tx_led(0); 
+  _set_mqtt_rx_led(2);
```

Comment:
Might be worth adding some local constants for the different led states? Make it much easier to follow what is going on.